libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -3,10 +3,14 @@
 
 .o-btn {
   &.is-neo {
-    @apply rounded-none font-inherit py-2;
+    @apply  h-10 min-w-10 rounded-none font-inherit py-2;
```

Comment:
this `h-10 min-w-10`  unless these values are overwritten inside each variant this might change all the other variants of buttons. 

can you check using storybook that nothing changed?

thanks

Reply:
we do need to change all the variants though?

Reply:
for some reason, stories arent working for me, keep seeing errors

Reply:
@hassnian